<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QWidget" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f0f0f0;
}

QPushButton {
    background-color: #4682b4;
    border: 1px solid #191970;
    color: white;
    border-radius: 10px;
    padding: 5px;
    font: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #191970;
}

QLabel {
    color: #000000;
    font: bold;
    font-size: 14px;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #4682b4;
    border-radius: 5px;
    padding: 2px;
}

QComboBox {
    background-color: #ffffff;
    border: 1px solid #4682b4;
    border-radius: 5px;
    padding: 2px;
    color: #000000;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    border: 1px solid #4682b4;
    selection-background-color: #4682b4;
    selection-color: white;
}

QComboBox::drop-down {
    border-left: 1px solid #4682b4;
    width: 20px;
}

QComboBox::down-arrow {
    image: url(:resources\\images\\triangle.png); 
    width: 12px; 
    height: 12px; 
    margin-top: 4px; 
}

QTabBar::tab {
    background: #1f5e8d; 
    border: 1px solid #191970;
    color: white;
    padding: 5px;
    border-radius: 5px;
}

QTabBar::tab:selected {
    background: #191970;
}

QTableView {
    gridline-color: #4682b4;
    selection-background-color: #f0f0f0;
    selection-color: black;
    background-color: #ffffff; 
}

QListView {
    font-size: 14px; 
}

QHeaderView {
    background-color: #ffffff;
}

QHeaderView::section {
    background-color: #4682b4;
    color: white;
    padding: 4px;
    border: 1px solid #191970;
}

QLineEdit { 
	font-size: 14px; 
}

QComboBox { 
	font-size: 14px; 
	padding: 5px; 
} 

QComboBox QAbstractItemView { 
	font-size: 14px;  
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="MainTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="profileTab">
      <attribute name="title">
       <string>профиль</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTabWidget" name="journalTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="journalTab">
          <attribute name="title">
           <string>журнал</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTableView" name="journalTableView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="resTab">
          <attribute name="title">
           <string>итог</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QTableView" name="resTableView_2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>40</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="groupLabel">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>класс </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>ФИО</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="avgGradeLabel">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>средний балл</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scheduleTab">
      <attribute name="title">
       <string>расписание</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="scheduleTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
